config.env.py

```diff
@@ -17,6 +17,9 @@
     'post_logout_redirect_uris': [os.environ.get('QUOTEFAULT_OIDC_LOGOUT_REDIRECT_URI', 'https://quotefault.csh.rit.edu/logout')]
 }
 
+# May fix SQLAlchemy losing connection to the database 
+SQLALCHEMY_POOL_RECYCLE = 500
```

Comment:
```suggestion
SQLALCHEMY_POOL_RECYCLE = os.environ.get("SQLALCHEMY_POOL_RECYCLE", default=500)
```

You mentioned making an env var, this is how we pull from the environment.